Sources/BadondeCore/ConfigurationStore.swift

```diff
@@ -110,3 +109,4 @@ private extension Encodable {
 		try data.write(to: url)
 	}
 }
+
```

Comment:
<!--
  1 failure:  Files should have...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/BadondeCore/ConfigurationStore.swift;
  Line: 112;
-->
- :no_entry_sign: Files should have a single trailing newline. (`trailing_newline`)